pandas/tests/frame/test_constructors.py

```diff
@@ -2457,6 +2457,32 @@ def test_constructor_list_str_na(self, string_dtype):
         expected = DataFrame({"A": ["1.0", "2.0", None]}, dtype=object)
         tm.assert_frame_equal(result, expected)
 
+    def test_constructor_large_size_frame(self):
```

Comment:
ah sorry, missed the GH reference - could you add a link to the issue https://github.com/pandas-dev/pandas/issues/50708 as a comment please?